spyder/plugins/ipythonconsole/utils/client.py

```diff
@@ -129,3 +129,12 @@ def tunnel_to_kernel(
             )
         )
         self.ip = "127.0.0.1"  # Tunneled to localhost
+
+    @staticmethod
+    def _split_shh_address(address):
+        """Split ssh address into host and port."""
+        user_host, _, port = address.partition(':')
+        user, _, host = user_host.rpartition('@')
+        return {'username': user if user else None,
+                'host': host if host else None,
+                'port': int(port) if port else None}
```

Comment:
```suggestion
        return {
            'username': user if user else None,
            'host': host if host else None,
            'port': int(port) if port else None
        }
```

Improve code formatting.